src/Playwright.Tests/BrowserTypeConnectTests.cs

```diff
@@ -113,5 +151,64 @@ public void ShouldTerminateNetworkWaiters()
         public void ShouldRespectSelectors()
         {
         }
+
+
+        private class BrowserServer
+        {
+            public Process Process { get; set; }
+            public string WSEndpoint { get; set; }
+        }
+
+        private static string GetExecutablePath()
```

Comment:
Why copy-paste this code and not run Program?

Reply:
Since, serverEnpoint is always changing we need to return that server endpoint & use in tests. 
If I start using Program I won't be able to get server endpoint in tests. Hence, had to do this.